### PR TITLE
AnalyzerWaveform: Fix commented out code

### PR DESCRIPTION
**Pull request by uklotzde**
The commented out code didn't compile.

TODO: The code does not contain any references how the frequency bands have been chosen nor why selected filters are used.

### CODE REVIEW

File: src/analyzer/analyzerwaveform.cpp [hunk 1]
```diff
@@ -12,6 +12,10 @@ namespace {
 
 mixxx::Logger kLogger("AnalyzerWaveform");
 
+constexpr double LOW_MID_FREQ_HZ = 600.0;
+
+constexpr double MID_HIGH_FREQ_HZ = 4000.0;
```
Comment (Swiftb0y) [inline src/analyzer/analyzerwaveform.cpp:17]:
```suggestion
constexpr double kLowMidFreqHz = 600.0;

constexpr double kMidHighFreqHz = 4000.0;
```

**Review by Swiftb0y — APPROVED** [review]
LGTM, but I'd appreciate if you'd use the naming scheme for the sake of consistency.

File: src/analyzer/analyzerwaveform.cpp (hunk 1 above)
Comment (uklotzde) [reply src/analyzer/analyzerwaveform.cpp:17]: I was just procrastinating while doing Rust development in parallel :rofl:

**Review by Swiftb0y — APPROVED** [review]
Thank you. I'd still like to wait with merging until CI passes. Unfortunately that seems to be broken right now because azure...

**Review by daschuer — APPROVED** [review]
All green now. LGTM. Thank you.

**Pull request merged**